### PR TITLE
UefiPayloadPkg: Enable Boot Logo

### PR DESCRIPTION
**Pull request by Sean-StarLabs**

### CODE REVIEW

File: UefiPayloadPkg/UefiPayloadPkg.dsc [hunk 1]
```diff
@@ -419,6 +425,7 @@
 
   gUefiPayloadPkgTokenSpaceGuid.PcdDispatchModuleAbove4GMemory|$(ABOVE_4G_MEMORY)
   gUefiPayloadPkgTokenSpaceGuid.PcdBootManagerEscape|$(BOOT_MANAGER_ESCAPE)
+
```
Comment (paulmenzel) [inline UefiPayloadPkg/UefiPayloadPkg.dsc:428]: Unrelated?

Comment (Sean-StarLabs) [reply UefiPayloadPkg/UefiPayloadPkg.dsc:428]: Uncrustify to pass the CI

File: UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c [hunk 2]
```diff
@@ -252,6 +252,11 @@ PlatformBootManagerAfterConsole (
   Black.Blue = Black.Green = Black.Red = Black.Reserved = 0;
   White.Blue = White.Green = White.Red = White.Reserved = 0xFF;
 
+  if (&gEdkiiPlatformLogoProtocolGuid) {
```
Comment (gguo11837463) [inline UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c:255]: The if condition looks like will always true.

File: UefiPayloadPkg/UefiPayloadPkg.fdf [hunk 3]
```diff
@@ -245,6 +245,8 @@ INF MdeModulePkg/Bus/Usb/UsbMouseDxe/UsbMouseDxe.inf
 # ACPI Support
 #
 INF  MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTableDxe.inf
+INF  MdeModulePkg/Universal/Acpi/AcpiPlatformDxe/AcpiPlatformDxe.inf
```
Comment (gguo11837463) [inline UefiPayloadPkg/UefiPayloadPkg.fdf:248]: It may miss macro !if $(BOOTSPLASH_IMAGE)

**Comment by Sean-StarLabs** [comment]
Should be good now :)

**Comment by LeanSheng** [comment]
Hi Sean could you add the commit message review by gua.guo@intel.com?
